server/sonar-server/src/main/java/org/sonar/server/ce/ws/TaskAction.java

```diff
@@ -120,7 +122,12 @@ public void handle(Request wsRequest, Response wsResponse) throws Exception {
 
   private void checkPermission(Optional<ComponentDto> component) {
     if (component.isPresent()) {
-      userSession.checkComponentPermission(SCAN_EXECUTION, component.get());
+      if (!userSession.hasOrganizationPermission(component.get().getOrganizationUuid(), GlobalPermissions.SYSTEM_ADMIN) &&
+        !userSession.hasOrganizationPermission(component.get().getOrganizationUuid(), GlobalPermissions.SCAN_EXECUTION) &&
```

Comment:
`SCAN_EXECUTION` is already statically imported (for the next line), let's use it! (and statically import  `SYSTEM_ADMIN` for consistency)